base/gfspclient/helper_test.go

```diff
@@ -145,6 +145,31 @@ func (*mockApproverServer) GfSpAskApproval(ctx context.Context, req *gfspserver.
 
 type mockAuthenticatorServer struct{}
 
+func (s mockAuthenticatorServer) GetAuthKeyV2(ctx context.Context, request *gfspserver.GetAuthKeyV2Request) (*gfspserver.GetAuthKeyV2Response, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
Fill in this part after gfspserver server part got implemented?

Reply:
fixed